blockchain/blockchain.go

```diff
@@ -353,10 +353,12 @@ func (bc *blockchain) startExistingBlockchain(recoveryHeight uint64) error {
 	}
 	ws, err := bc.sf.NewWorkingSet()
 	if err != nil {
-		return errors.Wrap(err, "Failed to obtain working set from state factory")
+		return errors.Wrap(err, "failed to obtain working set from state factory")
 	}
 	// If restarting factory from fresh db, first create creator's state
 	if startHeight == 0 {
+		actions := loadGenesisData(bc.config)
```

Comment:
why need this? i don't see Gen.CreatorPubKey being used?

Reply:
It is used below:
Gen.CreatorAddr(bc.ChainID())